salt/metalk8s/repo/installed.sls

```diff
@@ -55,6 +55,7 @@ Install repositories manifest:
         package_path: /{{ repo.relative_path }}
         image_path: '/images/'
         nginx_confd_path: {{ repo.config.directory }}
+        probe_host: {{ grains['metalk8s']['control_plane_ip'] }}
```

Comment:
Nitpick: most often, the subscription syntax is not useful in Jinja, doing `grains.metalk8s.control_plane_ip` is perfectly valid (and slightly more readable IMHO).

If you use it, at least make it efficient: `grains['metalk8s.control_plane_ip']` ;)

Reply:
Yup, I know, but I did write it this way for consistency across the project.